package.json

```diff
@@ -68,7 +68,7 @@
     "@npcz/magic": "^1.3.11",
     "@octokit/core": "^3.5.1",
     "@octokit/plugin-throttling": "^3.5.2",
-    "@octokit/rest": "^18.11.1",
+    "@octokit/rest": "18.0.0",
```

Comment:
❓

Reply:
https://github.com/iterative/cml/pull/839#issue-797145581

Reply:
🤦🏼‍♂️ 👍🏼

Reply:
https://github.com/octokit/rest.js/issues/130
I opened a ticket asking why the docs stated that the restart was available in docs but not in code